.circleci/config.yml

```diff
@@ -1,5 +1,12 @@
 version: 2
 jobs:
+  automated-tests:
+    docker:
+      - image: colibris/tox
+    steps:
+      - checkout
+      - run: tox --workdir=/tmp/tox --recreate
+
```

Comment:
We should also run some code style checkers.

Reply:
We surely do: https://github.com/colibris-framework/colibris/pull/29